### PR TITLE
Allow unit testing with PHP 7.2

### PR DESCRIPTION
**Pull request by AndreasMalecki**
This small fix will avoid errors with PHP 7.2 in unit tests caused by the following ini_set.

### CODE REVIEW

**Comment by Shardj** [comment]
Hi @AndreasMalecki, sorry I didn't get back to you sooner, I've been away. I can't see any issue with this really, I don't use this functionality myself though so when I run it $_defaultOptionsSet and self::$_unitTestEnabled are always false so it makes no difference to me.

However I don't want to make the change without understanding why it's needed. Could you post the error you're getting and a short explanation as to why the change is needed? From what I can tell, if you've set the session to say this runtime is a unit test then you don't want the default ini_set's to be run?

**Comment by SvenRtbg** [comment]
When running inside unit tests, and if you are basically initializing the whole application, Zend will also initialize the session settings that are configured. This will inevitably call `ini_set()` for the session settings, and this will trigger a warning that these settings cannot be changed anymore because headers are already sent.

Zend-Session has this `$_unitTestEnabled` property in order to allow testing session stuff without really calling PHP session functions, so you set this property to true in the test bootstrap or `setUp()` and can do session related activities without starting the real session - because on command line, doing real sessions is weird. Zend-Session acts as a wrapper around the core PHP session functions, and without calling them, no $_SESSION will be populated, but you are not supposed to access that in the first place, but should go via the relevant Zend_Session_Namespace object, which will continue to work in this scenario.

So TLDR: Calling `ini_set()` will trigger a warning, and the supposed setup in CLI tests is to set `$_unitTestEnabled` to true in order to prevent using the real PHP session.
